server/xpub-model/entities/manuscript/index.test.js

```diff
@@ -178,28 +263,45 @@ describe('Manuscript', () => {
       ).rejects.toThrow('Manuscript not found'))
   })
 
+  describe('refresh()', () => {
+    it('refreshes manuscript', async () => {
+      await testOverwritingWithOldData(userId, true)
```

Comment:
I'm not a fan of having tests like this. It makes it difficult to see what's being tested and what's being asserted on. I use the tests a lot like documentation, and extracting tests out like this makes it difficult to see how to call a function and what the behaviour of it is.

I'm happy extracting mocks and setup code to separate functions, but the invocation of the unit under test and the assertions should be more visible for each test.

If we really want to make our own DSL for the tests to make the more DRY, we'll want to make sure the tests are still readable.